### ISSUE TITLE
Outlook 2010: Unable to jump to quoted text despite NVDA announcing a heading level 1

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#5518]
In Outlook 2010, when reading the body of an HTML email the text that sits directly below the authors last reply (i.e. the text that was previously quoted by them) is reported by NVDA as having a heading at level 1. However, pressing H reports that there are no visible headings within the document.

**Comment by jcsteh** [comment]
This is similar to #5186. However, this is almost the reverse because this probably really shouldn't be treated as a heading. It is one example of where outline level is used for something *other* than headings. The question is whether this is valid or just a hack. It'd also be good to know what other screen readers do with quoted text in Outlook.

**Comment by dkager** [comment]
Based on Outlook 2016:
* SuperNova doesn't report any headings at all.
* NVDA speaks all the fields of the quoted message as headings, but curiously only indicates the first one (From) as a heading in braille. I guess this whole block has the same outline.
* Stating the obvious, but I can reproduce this issue.

**Comment by jcsteh** [comment]
> SuperNova doesn't report any headings at all.

Does it report the fact that it's quoted, though? For that matter, does it report anything at all for that text apart from the text itself?

**Comment by dkager** [comment]
With default settings, SuperNova just reported the text. It does report HTML headings in e-mails, though. I’m also not sure if comparing yourself to SuperNova where Office is concerned is the best idea, now that I think on it…

**Comment by leonardder** [comment]
@dkager: Is this still reproducible? I wonder whether this is somehow fixed by the UIA implementation.

**Comment by dkager** [comment]
I don't see any headings for quoted messages in recent versions of Office 365. That is with the default reply style.

**Comment by Adriani90** [comment]
@elliott94, @dkager could you please send an example E-Mail for testing? I could test if this is the case in MS Word 2010.

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since many years, and from previous comments I see that this is not reproducible in Office 365. I am closing this as abandoned since support for Office 2010 ended completely in 2020 and for 2013 in April 2023. I don't expect any further accessibility updates to these Office versions.
